padertorch/utils.py

```diff
@@ -142,6 +143,20 @@ def to_numpy(array, detach: bool = False, copy: bool = False) -> np.ndarray:
         if detach:
             array = array.detach()
 
+    if isinstance(array, torch.Tensor) and array.is_sparse:
+        if force_dense:
+            array = array.to_dense()
+        else:
+            raise NotImplementedError
```

Comment:
Why do you raise a NotImplementedError? Will Sparce Tensors not be converted to scipy? Or are the ambiguities?